aws/credentials/stscreds/web_identity_provider.go

```diff
@@ -52,6 +52,11 @@ type WebIdentityRoleProvider struct {
 	credentials.Expiry
 	PolicyArns []*sts.PolicyDescriptorType
 
+	// Expiry duration of the STS credentials. Will be truncated to seconds when used to assume the role.
+	// If unset, the assumed role will use AssumeRoleWithWebIdentity's default expiry duration. See
+	// https://docs.aws.amazon.com/sdk-for-go/api/service/sts/#STS.AssumeRoleWithWebIdentity for more information.
+	Duration time.Duration
+
 	client       stsiface.STSAPI
 	ExpiryWindow time.Duration
```

Comment:
Can you add a comment on what's the difference between Duration and ExpiryWindow? 

Both are exported members but seem to have similar purpose and might be confused.

Reply:
Sure will update with docs.